--- interface\camp_repression_ledger.gui
+++ interface\camp_repression_ledger.gui
@@ -12,31 +12,31 @@
 		instantTextBoxType = { name = "camp_ui_title" position = { x = 26 y = 18 } font = "hoi_36header" text = "camp_ui_title" maxWidth = 740 maxHeight = 42 format = left fixedsize = yes alwaysTransparent = yes  }
 		instantTextBoxType = { name = "camp_ui_institution" position = { x = 26 y = 62 } font = "hoi_18mbs" text = "camp_ui_institution" maxWidth = 740 maxHeight = 24 format = left fixedsize = yes alwaysTransparent = yes  }
 		buttonType = { name = "camp_gui_close_window" position = { x = 912 y = 18 } spriteType = "GFX_closebutton" buttonFont = "hoi_16mbs" buttonText = "" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "camp_gui_close_window_tt" }
-		iconType = { name = "camp_ui_navigation_background" position = { x = 16 y = 108 } size = { x = 180 y = 474 } quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes  }
+		containerWindowType = { name = "camp_ui_navigation_background" position = { x = 16 y = 108 } size = { width = 180 height = 474 } background = { name = "background" quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes } }
 		buttonType = { name = "camp_gui_tab_overview" position = { x = 26 y = 128 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "repression_ledger_tab_overview_tt" }
-		iconType = { name = "repression_ledger_tab_overview_mark" position = { x = 26 y = 128 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
+		buttonType = { name = "repression_ledger_tab_overview_mark" position = { x = 26 y = 128 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
 		instantTextBoxType = { name = "camp_ui_nav_overview" position = { x = 36 y = 141 } font = "hoi_18b" text = "camp_ui_nav_overview" maxWidth = 140 maxHeight = 24 format = center fixedsize = yes alwaysTransparent = yes  }
 		buttonType = { name = "camp_gui_tab_state_pools" position = { x = 26 y = 187 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "repression_ledger_tab_state_pools_tt" }
-		iconType = { name = "repression_ledger_tab_state_pools_mark" position = { x = 26 y = 187 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
+		buttonType = { name = "repression_ledger_tab_state_pools_mark" position = { x = 26 y = 187 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
 		instantTextBoxType = { name = "camp_ui_nav_state_pools" position = { x = 36 y = 200 } font = "hoi_18b" text = "camp_ui_nav_state_pools" maxWidth = 140 maxHeight = 24 format = center fixedsize = yes alwaysTransparent = yes  }
 		buttonType = { name = "camp_gui_tab_sites" position = { x = 26 y = 246 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "repression_ledger_tab_sites_tt" }
-		iconType = { name = "repression_ledger_tab_sites_mark" position = { x = 26 y = 246 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
+		buttonType = { name = "repression_ledger_tab_sites_mark" position = { x = 26 y = 246 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
 		instantTextBoxType = { name = "camp_ui_nav_sites" position = { x = 36 y = 259 } font = "hoi_18b" text = "camp_ui_nav_sites" maxWidth = 140 maxHeight = 24 format = center fixedsize = yes alwaysTransparent = yes  }
 		buttonType = { name = "camp_gui_tab_country" position = { x = 26 y = 305 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "repression_ledger_tab_country_tt" }
-		iconType = { name = "repression_ledger_tab_country_mark" position = { x = 26 y = 305 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
+		buttonType = { name = "repression_ledger_tab_country_mark" position = { x = 26 y = 305 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
 		instantTextBoxType = { name = "camp_ui_nav_country" position = { x = 36 y = 318 } font = "hoi_18b" text = "camp_ui_nav_country" maxWidth = 140 maxHeight = 24 format = center fixedsize = yes alwaysTransparent = yes  }
 		buttonType = { name = "camp_gui_tab_discovery" position = { x = 26 y = 364 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "repression_ledger_tab_discovery_tt" }
-		iconType = { name = "repression_ledger_tab_discovery_mark" position = { x = 26 y = 364 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
+		buttonType = { name = "repression_ledger_tab_discovery_mark" position = { x = 26 y = 364 } size = { x = 160 y = 44 } quadTextureSprite = "GFX_tiled_window_small_selectable" alwaysTransparent = yes frame = 2 }
 		instantTextBoxType = { name = "camp_ui_nav_discovery" position = { x = 36 y = 377 } font = "hoi_18b" text = "camp_ui_nav_discovery" maxWidth = 140 maxHeight = 24 format = center fixedsize = yes alwaysTransparent = yes  }
 		instantTextBoxType = { name = "camp_ui_navigation_note" position = { x = 32 y = 478 } font = "hoi_16mbs" text = "camp_ui_navigation_note" maxWidth = 148 maxHeight = 78 format = left fixedsize = yes alwaysTransparent = yes  }
 		containerWindowType = { name = "repression_ledger_overview_panel" position = { x = 218 y = 116 } size = { width = 708 height = 460 }
 			instantTextBoxType = { name = "camp_ui_situation_heading" position = { x = 0 y = 0 } font = "hoi_24header" text = "camp_ui_situation_heading" maxWidth = 680 maxHeight = 32 format = left fixedsize = yes alwaysTransparent = yes  }
-			iconType = { name = "camp_ui_harm_surface" position = { x = 0 y = 44 } size = { x = 708 y = 154 } quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes  }
+			containerWindowType = { name = "camp_ui_harm_surface" position = { x = 0 y = 44 } size = { width = 708 height = 154 } background = { name = "background" quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes } }
 			instantTextBoxType = { name = "camp_ui_harm_label" position = { x = 24 y = 61 } font = "hoi_20b" text = "camp_ui_harm_label" maxWidth = 640 maxHeight = 26 format = left fixedsize = yes alwaysTransparent = yes  }
 			instantTextBoxType = { name = "camp_ui_harm_value" position = { x = 24 y = 91 } font = "hoi_36header" text = "camp_ui_harm_value" maxWidth = 640 maxHeight = 46 format = left fixedsize = yes alwaysTransparent = yes pdx_tooltip = "camp_ui_harm_tt" }
 			instantTextBoxType = { name = "camp_ui_harm_help" position = { x = 24 y = 144 } font = "hoi_18mbs" text = "camp_ui_harm_help" maxWidth = 650 maxHeight = 44 format = left fixedsize = yes alwaysTransparent = yes  }
-			iconType = { name = "camp_ui_strain_surface" position = { x = 0 y = 212 } size = { x = 348 y = 172 } quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes  }
-			iconType = { name = "camp_ui_evidence_surface" position = { x = 360 y = 212 } size = { x = 348 y = 172 } quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes  }
+			containerWindowType = { name = "camp_ui_strain_surface" position = { x = 0 y = 212 } size = { width = 344 height = 172 } background = { name = "background" quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes } }
+			containerWindowType = { name = "camp_ui_evidence_surface" position = { x = 360 y = 212 } size = { width = 348 height = 172 } background = { name = "background" quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes } }
 			instantTextBoxType = { name = "camp_ui_strain_label" position = { x = 22 y = 230 } font = "hoi_20b" text = "camp_ui_strain_label" maxWidth = 300 maxHeight = 25 format = left fixedsize = yes alwaysTransparent = yes  }
 			instantTextBoxType = { name = "camp_ui_strain_value" position = { x = 22 y = 263 } font = "hoi_24header" text = "camp_ui_strain_value" maxWidth = 300 maxHeight = 36 format = left fixedsize = yes alwaysTransparent = yes pdx_tooltip = "camp_ui_strain_tt" }
 			instantTextBoxType = { name = "camp_ui_strain_help" position = { x = 22 y = 309 } font = "hoi_16mbs" text = "camp_ui_strain_help" maxWidth = 300 maxHeight = 56 format = left fixedsize = yes alwaysTransparent = yes  }
@@ -146,13 +146,13 @@
 		containerWindowType = { name = "repression_ledger_discovery_panel" position = { x = 218 y = 116 } size = { width = 708 height = 460 }
 			instantTextBoxType = { name = "camp_ui_accountability_heading" position = { x = 0 y = 0 } font = "hoi_24header" text = "camp_ui_accountability_heading" maxWidth = 688 maxHeight = 30 format = left fixedsize = yes alwaysTransparent = yes  }
 			instantTextBoxType = { name = "camp_ui_accountability_intro" position = { x = 0 y = 40 } font = "hoi_18mbs" text = "camp_ui_accountability_intro" maxWidth = 688 maxHeight = 44 format = left fixedsize = yes alwaysTransparent = yes  }
-			iconType = { name = "camp_ui_record_exposure" position = { x = 0 y = 100 } size = { x = 708 y = 104 } quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes  }
+			containerWindowType = { name = "camp_ui_record_exposure" position = { x = 0 y = 100 } size = { width = 708 height = 104 } background = { name = "background" quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes } }
 			instantTextBoxType = { name = "camp_ui_record_exposure_label" position = { x = 22 y = 114 } font = "hoi_20b" text = "camp_ui_record_exposure_label" maxWidth = 194 maxHeight = 29 format = left fixedsize = yes alwaysTransparent = yes  }
 			instantTextBoxType = { name = "camp_ui_record_exposure_value" position = { x = 234 y = 114 } font = "hoi_18mbs" text = "camp_ui_record_exposure_value" maxWidth = 450 maxHeight = 68 format = left fixedsize = yes alwaysTransparent = yes pdx_tooltip = "repression_discovery_status_card_tt" }
-			iconType = { name = "camp_ui_record_losses" position = { x = 0 y = 216 } size = { x = 708 y = 104 } quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes  }
+			containerWindowType = { name = "camp_ui_record_losses" position = { x = 0 y = 216 } size = { width = 708 height = 104 } background = { name = "background" quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes } }
 			instantTextBoxType = { name = "camp_ui_record_losses_label" position = { x = 22 y = 230 } font = "hoi_20b" text = "camp_ui_record_losses_label" maxWidth = 194 maxHeight = 29 format = left fixedsize = yes alwaysTransparent = yes  }
 			instantTextBoxType = { name = "camp_ui_record_losses_value" position = { x = 234 y = 230 } font = "hoi_18mbs" text = "camp_ui_record_losses_value" maxWidth = 450 maxHeight = 68 format = left fixedsize = yes alwaysTransparent = yes pdx_tooltip = "repression_deaths_card_tt" }
-			iconType = { name = "camp_ui_record_closure" position = { x = 0 y = 332 } size = { x = 708 y = 104 } quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes  }
+			containerWindowType = { name = "camp_ui_record_closure" position = { x = 0 y = 332 } size = { width = 708 height = 104 } background = { name = "background" quadTextureSprite = "GFX_tiled_window_small" alwaysTransparent = yes } }
 			instantTextBoxType = { name = "camp_ui_record_closure_label" position = { x = 22 y = 346 } font = "hoi_20b" text = "camp_ui_record_closure_label" maxWidth = 194 maxHeight = 29 format = left fixedsize = yes alwaysTransparent = yes  }
 			instantTextBoxType = { name = "camp_ui_record_closure_value" position = { x = 234 y = 346 } font = "hoi_18mbs" text = "camp_ui_record_closure_value" maxWidth = 450 maxHeight = 68 format = left fixedsize = yes alwaysTransparent = yes pdx_tooltip = "repression_reform_route_card_tt" }
 		}
@@ -162,7 +162,7 @@
 		name = "camp_ui_pool_row"
 		size = { width = 274 height = 52 }
 		buttonType = { name = "camp_ui_pool_select" position = { x = 0 y = 0 } size = { x = 274 y = 48 } quadTextureSprite = "GFX_tiled_window_small" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "camp_ui_location_select_tt" }
-		iconType = { name = "camp_ui_pool_selected" position = { x = 0 y = 0 } size = { x = 274 y = 48 } quadTextureSprite = "GFX_tiled_window_small_selectable" frame = 2 alwaysTransparent = yes }
+		buttonType = { name = "camp_ui_pool_selected" position = { x = 0 y = 0 } size = { x = 274 y = 48 } quadTextureSprite = "GFX_tiled_window_small_selectable" frame = 2 alwaysTransparent = yes }
 		instantTextBoxType = { name = "camp_ui_pool_name" position = { x = 12 y = 9 } font = "hoi_18mbs" text = "camp_ui_location_name" maxWidth = 250 maxHeight = 38 format = left fixedsize = yes alwaysTransparent = yes }
 	}
 
@@ -170,7 +170,7 @@
 		name = "camp_ui_site_row"
 		size = { width = 274 height = 52 }
 		buttonType = { name = "camp_ui_site_select" position = { x = 0 y = 0 } size = { x = 274 y = 48 } quadTextureSprite = "GFX_tiled_window_small" clicksound = decisions_ui_button oversound = ui_menu_over pdx_tooltip = "camp_ui_location_select_tt" }
-		iconType = { name = "camp_ui_site_selected" position = { x = 0 y = 0 } size = { x = 274 y = 48 } quadTextureSprite = "GFX_tiled_window_small_selectable" frame = 2 alwaysTransparent = yes }
+		buttonType = { name = "camp_ui_site_selected" position = { x = 0 y = 0 } size = { x = 274 y = 48 } quadTextureSprite = "GFX_tiled_window_small_selectable" frame = 2 alwaysTransparent = yes }
 		instantTextBoxType = { name = "camp_ui_site_name" position = { x = 12 y = 9 } font = "hoi_18mbs" text = "camp_ui_location_name" maxWidth = 250 maxHeight = 38 format = left fixedsize = yes alwaysTransparent = yes }
 	}
 }
