--- before
+++ after
@@ -303,6 +303,7 @@
 		state_target = any
 		on_map_mode = map_and_decisions_view
 		cost = 0
+		ai_hint_pp_cost = 30
 		custom_cost_trigger = { camp_rework_can_pay_site_labor = { STATE = FROM } }
 		custom_cost_text = camp_cost_site_labor
 		days_re_enable = constant:camp_rework_timing.standard_action_days
@@ -420,6 +421,7 @@
 		state_target = any
 		on_map_mode = map_and_decisions_view
 		cost = 0
+		ai_hint_pp_cost = 30
 		custom_cost_trigger = { camp_rework_can_pay_site_labor = { STATE = FROM } }
 		custom_cost_text = camp_cost_site_labor
 		days_re_enable = constant:camp_rework_timing.standard_action_days
@@ -674,6 +676,7 @@
 		state_target = any
 		on_map_mode = map_and_decisions_view
 		cost = 0
+		ai_hint_pp_cost = 60
 		custom_cost_trigger = {
 			camp_rework_can_pay_site_restricted = { STATE = FROM }
 			OR = {
@@ -768,6 +771,7 @@
 		state_target = any
 		on_map_mode = map_and_decisions_view
 		cost = 0
+		ai_hint_pp_cost = 25
 		custom_cost_trigger = { camp_rework_can_pay_site_evidence = { STATE = FROM } }
 		custom_cost_text = camp_cost_site_evidence
 		days_re_enable = constant:camp_rework_timing.retreat_days
@@ -804,6 +808,7 @@
 		state_target = any
 		on_map_mode = map_and_decisions_view
 		cost = 0
+		ai_hint_pp_cost = 45
 		custom_cost_trigger = { camp_rework_can_pay_site_inspect = { STATE = FROM } }
 		custom_cost_text = camp_cost_site_inspect
 		days_re_enable = constant:camp_rework_timing.idea_medium_days
@@ -845,6 +850,7 @@
 		state_target = any
 		on_map_mode = map_and_decisions_view
 		cost = 0
+		ai_hint_pp_cost = 60
 		custom_cost_trigger = { camp_rework_can_pay_site_dismantle = { STATE = FROM } }
 		custom_cost_text = camp_cost_site_dismantle
 		days_remove = constant:camp_rework_timing.dismantle_days
